CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Add Thanos compact --retention.default flag, for configuring storage bucket retention period.
 - Removes support for multiple units in duration. For example: 1m0s won't work, while 1m will work.
 - Adds support for y,w,d time units 
+- Use the configuration of bucket in yaml format.
```

Comment:
Can we move to something like:

Removes support of those flags
```
--gcs-bucket=<bucket>
 --s3.bucket=<bucket>     
--s3.endpoint=<api-url> 
--s3.access-key=<key>    
--s3.insecure     
--s3.signature-version2  
--s3.encrypt-sse     
--gcs-backup-bucket=<bucket>  	
--s3-backup-bucket=<bucket>  
```
instead added `--objstore.config`